src/test/java/atlastests/PredictionStepDefs.java

```diff
@@ -276,6 +291,11 @@ public void clickToUtilitiesButtonInPrediction() {
         $(By.xpath("//*[@class='tabs__header']/span[3]")).click();
     }
 
+    @When("^click to Utilities button in Prediction without ee$")
+    public void clickToUtilitiesButtonInPredictioWithoutEEn() {
+        $(By.xpath("//*[@class='tabs__header']/span[2]")).click();
```

Comment:
two same methods with a different name, please, remove one of them

Reply:
here it's also would be better to use something like this $$("#tabs__header).find(Condition.matchesText("Utilities")).click();